eFFORT/hybrid/hybrid_model.py

```diff
@@ -69,7 +77,7 @@ def calculate_weight(self, x: numpy.array, hybrid_weights: numpy.histogramdd) ->
         syntax, assuming you stored the result of :func:`~hybrid.Hybrid.generate_hybrid_weights` in the variable
         hybrid_weights, and your data is stored in the pandas.DataFrame df:
 
-            >>> # df['hybrid_weight'] = calculate_weight(df[['El_B', 'q2', 'mX']], hybrid_weights)
+            >>> # df['hybrid_weight'] = calculate_weight(df[['El_B', 'q2', 'mX']].values, hybrid_weights)
```

Comment:
Not part of your changes, but is the hash after the prompt there for a reason? I see no reason why the example should be a commented out line?

Nice that you added the `.values` :+1: 

Btw, I didn't know that sphinx renders interactive prompts automatically if you include the prompt, I just found out when I checked if the `>>>` would work, I had only used code blocks before.

Reply:
Yes the hash is there in case you run the doctest, as this would otherwise fail.